components/testmonitor/actions/find-issue/find-issue.mjs

```diff
@@ -0,0 +1,38 @@
+import common from "../common/base.mjs";
+
+export default {
+  ...common,
+  key: "testmonitor-find-issue",
+  name: "Find an Issue",
+  description: "Retrieve an issue using its identifier. [See the docs here](https://docs.testmonitor.com/#tag/Issues/operation/GetIssue)",
```

Comment:
Observation:
I was wondering if all these actions actually are refering to kind of a search based on somo text query and filters like you can see in their docs [here](https://docs.testmonitor.com/#tag/Issues/operation/GetIssueCollection)